mod/01_rating/set/type/metric_value.rb

```diff
@@ -345,26 +345,22 @@ def find_potential_sources company, metric
   end
 
   def set_hidden_args args
-    if !args[:source]
```

Comment:
this was for old flow. While users click add new metric value from the source page, it will have source in the args. It should still show the metric subcard field here.